doofinder.php

```diff
@@ -289,7 +289,7 @@ public function getContent()
             'tab_module' => $this->tab,
             'module_name' => $this->name,
         ];
-        $skipurl = $this->context->link->getAdminLink('AdminModules', true) . '?' . http_build_query($skip_url_params);
+        $skipurl = $this->context->link->getAdminLink('AdminModules', true) . '&' . http_build_query($skip_url_params);
```

Comment:
I'd suggest checking if the current link has a '?' or not, to decide if the needed guard to include is '&' or '?'